### PR TITLE
MBS-10726: Use "medium", not "disc", as a generic term in RE buttons

### PR DESCRIPTION
**Pull request by reosarevok**
### Implement MBS-10726

These can apply to mediums that are not discs. The pregap / data track strings only apply to actual discs, so not changing those for now.

### CODE REVIEW

**Comment by yvanzo** [comment]
@brainzbot retest this please

File: root/release/edit/tracklist.tt [hunk 1]
```diff
@@ -285,7 +285,7 @@
     <table class="advanced-format">
       <tr>
         <td class="icon">
-          <button type="button" class="icon collapse-disc" data-click="toggleMedium" data-bind="css: { 'expand-disc': collapsed, 'collapse-disc': !collapsed() }, attr: { title: collapsed() ? '[% l('Expand Disc') %]' : '[% l('Collapse Disc') %]' }"></button>
+          <button type="button" class="icon collapse-disc" data-click="toggleMedium" data-bind="css: { 'expand-disc': collapsed, 'collapse-disc': !collapsed() }, attr: { title: collapsed() ? '[% l('Expand medium') %]' : '[% l('Collapse medium') %]' }"></button>
```
Comment (yvanzo) [inline root/release/edit/tracklist.tt:288]: `collapse-disc`and `expand-disc` should probably be renamed too.

File: root/release/edit/tracklist.tt [hunk 2]
```diff
@@ -302,19 +302,19 @@
           </label>
           (<a href="[% c.uri_for('/doc/Release/Format') %]" target="_blank">[% l('help') %]</a>)
           <!-- ko if: release.mediums().length > 1 || name() -->
-            <label data-bind="attr: { for: 'disc-title-' + uniqueID }">[% l('Disc title:') %]</label>
+            <label data-bind="attr: { for: 'disc-title-' + uniqueID }">[% l('Medium title:') %]</label>
```
Comment (yvanzo) [inline root/release/edit/tracklist.tt:305]: The same with `disc-title-<id>`.

File: root/release/edit/tracklist.tt [hunk 3]
```diff
@@ -302,19 +302,19 @@
...
             [%~ l('A format is required. If you don’t know it, tick the “I don’t know” checkbox next to the format dropdown.') %]
           </p>
         </td>
 
         <td class="align-right icon" >
-          <button type="button" class="icon disc-down" title="[% l('Move disc down') %]" data-click="moveMediumDown"></button>
-          <button type="button" class="icon disc-up" title="[% l('Move disc up') %]" data-click="moveMediumUp"></button>
-          <button type="button" class="icon remove-item" title="[% l('Remove disc') %]" style="margin-left: 16px" data-click="removeMedium"></button>
+          <button type="button" class="icon disc-down" title="[% l('Move medium down') %]" data-click="moveMediumDown"></button>
+          <button type="button" class="icon disc-up" title="[% l('Move medium up') %]" data-click="moveMediumUp"></button>
```
Comment (yvanzo) [inline root/release/edit/tracklist.tt:316]: The same with `disc-down` and `disc-up`.

**Review by yvanzo — CHANGES_REQUESTED** [review]
Please add a commit to rename misleading identifiers too.

**Comment by reosarevok** [comment]
I thought I wouldn't touch those to avoid making this bigger, but I guess it is unlikely to break anything (scripts or whatnot), so I can make the change :)

**Comment by reosarevok** [comment]
I made the change, but properly then - I think I changed everything in the RE that should be medium but was disc :) Do let me know if you find something I missed.

**Review by yvanzo — APPROVED** [review]
Roughly tested with editing “The Downward Spiral” release from sample data: http://localhost:5000/release/0ce274e3-3b89-4d15-94ac-4f92e497ea60/edit

There are a few more occurrences of `disc` in code but that would be a much deeper change since we use it in URLs. Lasting occurrences can be changed later on.

File: root/release/edit/tracklist.tt [hunk 4]
```diff
@@ -226,32 +226,32 @@
...
-<div id="add-disc-dialog" style="display: none;" data-bind="with: $root.addDiscDialog, delegatedHandler: ['click', 'keydown']">
+<div id="add-medium-dialog" style="display: none;" data-bind="with: $root.addMediumDialog, delegatedHandler: ['click', 'keydown']">
   <p>[% l('To create a new tracklist, use an existing medium or import a disc from a CD stub, select the appropriate tab.') %]</p>
 
   <ul>
-    <li><a href="#add-disc-parser">[% l('Manual entry') %]</a></li>
-    <li><a href="#add-disc-medium">[% l('Existing medium') %]</a></li>
-    <li><a href="#add-disc-cdstub">[% l('CD stub import') %]</a></li>
+    <li><a href="#add-medium-parser">[% l('Manual entry') %]</a></li>
+    <li><a href="#add-medium-medium">[% l('Existing medium') %]</a></li>
```
Comment (mwiencek) [inline root/release/edit/tracklist.tt:234]: > add-medium-medium

Maybe we should rename this to `add-medium-existing` or something else now :D

**Review by mwiencek — APPROVED** [review]

**Pull request merged**